src/palladio/AttributeConversion.cpp

```diff
@@ -348,7 +348,8 @@ void ToHoudini::createAttributeHandles(bool useArrayTypes) {
 						handle = h;
 				}
 				else {
-					GA_RWHandleF h(mDetail->addFloatTuple(GA_ATTRIB_PRIMITIVE, utKey, hm.second.cardinality));
+					GA_RWHandleD h(mDetail->addFloatTuple(GA_ATTRIB_PRIMITIVE, utKey, hm.second.cardinality,
+						                                  GA_Defaults(0), nullptr, nullptr, GA_STORE_REAL64));
```

Comment:
The `addXTuple` functions take the `defaults` argument by reference. We could use this opportunity to extract `GA_Defaults(0)` into a namespace private const. There is one more occurrence.